source/events/test/test_media_events.py

```diff
@@ -55,7 +55,8 @@ def test_lambda_handler(self, patched_resource,
         with patch.object(media_events.EVENTS_TABLE, 'put_item', return_value={}):
             with patch.object(media_events.CLOUDWATCH_EVENTS_TABLE, 'put_item', return_value={}):
                 for event in mocked_events:
-                    media_events.lambda_handler(event, MagicMock())
+                    result = media_events.lambda_handler(event, MagicMock())
+                    self.assertTrue(result)
```

Comment:
I have added additional assertions (`put_item`), but asserting the return value of `lambda_handler` to be `True` was an actual indented assertion because, just like you said, I wanted to ensure the function reaches the end of the line without any exceptions thrown before that happens.